### PR TITLE
Re-arrange folder structure for the dart side

### PR DESCRIPTION
**Pull request by luanpotter**
This is just actually implementing [my proposal](https://github.com/luanpotter/audioplayers/pull/837) for file organization on the updated master branch.

image

### CODE REVIEW

**Review by spydon — APPROVED** [review]

File: lib/src/notifications/notification_service.dart [hunk 1]
```diff
@@ -34,6 +31,14 @@ class NotificationService {
     }
   }
 
+  final StreamController<PlayerControlCommand> _commandController =
+      StreamController<PlayerControlCommand>.broadcast();
+
+  /// Stream of remote player command send by native side
```
Comment (erickzanardo) [inline lib/src/notifications/notification_service.dart:37]: Not sure I fully understood this sentence, but it should be in the past I think
```suggestion
  /// Stream of remote player command sent by native side
```

File: lib/src/notifications/notification_service.dart [hunk 2]
```diff
@@ -34,6 +31,14 @@ class NotificationService {
...
+  ///
+  /// Events are sent user tap system remote control command.
```
Comment (erickzanardo) [inline lib/src/notifications/notification_service.dart:39]: This sentence is a little confusing as well, not sure I can suggest something as I am not sure I got it

**Review by erickzanardo — APPROVED** [review]
Couple of nits, but LGTM

**Comment by luanpotter** [comment]
@erickzanardo these are all existing comments, I am just moving to a new file

**Comment by erickzanardo** [comment]
> @erickzanardo these are all existing comments, I am just moving to a new file

Either way, no harm in improving then right?

**Comment by luanpotter** [comment]
Yeah but I don't know what they mean either...

**Comment by luanpotter** [comment]
Actually I think these should be changed, I think I know what they are now

**Pull request merged**